agent/agent.go

```diff
@@ -40,7 +40,8 @@ import (
 const hostAttribute = "evergreen.host"
 
 var (
-	shouldExitAttribute = fmt.Sprintf("%s.should_exit", hostAttribute)
+	shouldExitAttribute  = fmt.Sprintf("%s.should_exit", hostAttribute)
+	globalFilesToCleanup = []string{".gitconfig", ".git-credentials", ".netcr"}
```

Comment:
Not to do it in this ticket, but should this be an admin config in the future (like a future ticket work)?

Also, should we comment here that these files exist relative to some path and what path it is (so if someone has to add something not in the home directory, how would they use this constant to do so?)